cmd/snap-seccomp/main.go

```diff
@@ -388,6 +388,18 @@ var seccompResolver = map[string]uint64{
 	"NETLINK_RDMA":           C.NETLINK_RDMA,
 	"NETLINK_CRYPTO":         C.NETLINK_CRYPTO,
 	"NETLINK_INET_DIAG":      C.NETLINK_INET_DIAG, // synonymous with NETLINK_SOCK_DIAG
+
```

Comment:
Please run `go fmt` on this file.

Reply:
Thanks; the last time I wrote any Go was years ago, so I'd forgotten about `go fmt`.